File: core/src/main/java/com/graphhopper/routing/ev/AverageSlope.java
package com.graphhopper.routing.ev;

/**
 * Average elevation. Will be negated in reverse direction.
 */
public class AverageSlope {
    public static final String KEY = "average_slope";

    public static DecimalEncodedValue create() {
        return new DecimalEncodedValueImpl(KEY, 6, 0, 0.5,
                true, false, false);
    }
}

File: core/src/main/kotlin/com/graphhopper/routing/ev/AverageSlope.kt
package com.graphhopper.routing.ev

/**
 * Average elevation. Will be negated in reverse direction.
 */
object AverageSlope {
    const val KEY = "average_slope"

    @JvmStatic
    fun create(): DecimalEncodedValue = DecimalEncodedValueImpl(
        KEY, 5, 0.0, 1.0,
        true, false, false
    )
}
